src/backend/src/services/organizations.service.ts

```diff
@@ -53,4 +53,25 @@ export default class OrganizationsService {
 
     return newLinks;
   }
+
+  static async getAllUsefulLinks(submitter: User, organizationId: string) {
+    if (!(await userHasPermission(submitter.userId, organizationId, isAdmin)))
```

Comment:
We don't want this check because the get all endpoint will be used to display links on the homepage, which we want everyone to be able to see